src/GraphQL/GraphQLExtensions.cs

```diff
@@ -126,6 +159,13 @@ public static bool IsValidInterfaceFor(this IInterfaceGraphType iface, IObjectGr
             return true;
         }
 
+        /// <summary>
+        /// Returns a new instance of the specified graph type, using the specified resolver to
+        /// instantiate a new instance. Defaults to <see cref="Activator.CreateInstance(Type)"/>
+        /// if no <paramref name="resolve"/> parameter is specified. List and non-null graph
+        /// types are instantiated and their <see cref="IProvideResolvedType.ResolvedType"/> property
```

Comment:
```suggestion
        /// types are instantiated and their <see cref="IProvideResolvedType.ResolvedType"/>
```

It doubled up, strange.